client/src/app.c

```diff
@@ -394,6 +394,24 @@ void app_handleKeyRelease(int sc)
   }
 }
 
+bool app_receiveKeyboardTyping(void)
```

Comment:
`app_isOverlayMode` would be a more appropriate name for this.

Reply:
Fixed.